### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by FFY00**
* Updated `README.md` to european portuguese
  * Created `CONTRIBUTING.md` describing the text style

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,6 @@ Versão em português da DLang Tour
...
-Achou um error de digitação ou quer melhorar o conteúdo?
-Apenas clique em "edit" e envie um pull request para nós.
-Se você quer discutir uma ideia antes de alterar algo,
-não hesite em abrir uma [issue](https://github.com/dlang-tour/portuguese/issues).
-
-Você também pode estar procurando pelo [repositório base](https://github.com/dlang-tour)
-que hospeda o conteúdo.
+Para report algum erro, dar feedback ou apenas contactar a equipa basta criar
+um [Issue](https://github.com/dlang-tour/portuguese/issues) e para contribuir
+basta criar um [Pull Request](https://github.com/FFY00/portuguese/pulls).
```
Comment (wilzbach) [inline README.md]: Shouldn't this be dlang-tour?

Comment (FFY00) [reply README.md]: yes, sorry. haha

**Comment by wilzbach** [comment]
BTW if you know of who speak Portuguese and want to join the translation team on GitHub, so that they can review and merge your work, just let me know.

**Comment by FFY00** [comment]
I think @ljmf00 might be interested.

**Comment by wilzbach** [comment]
@FFY00 Invited!

**Pull request merged**